conf/defaults.ini

```diff
@@ -366,6 +366,9 @@ data_keys_cache_cleanup_interval = 1m
 
 #################################### Snapshots ###########################
 [snapshots]
+# set to false to remove snapshot functionality
```

Comment:
I think we want to add this new configuration also in [the docs](https://github.com/grafana/grafana/blob/main/docs/sources/setup-grafana/configure-grafana/_index.md) as well

Reply:
Hi @papagian , I made a separate [PR](https://github.com/grafana/grafana/pull/62110) for that